### ISSUE TITLE
Why was CatBoost excluded from the benchmark comparisons?

### ISSUE THREAD

**Issue by valeman** [jonaslandsgesell/ScoringBench#2]
Why was CatBoost excluded from the comparisons? Its absence from Table 2 especially undermines confidence in the benchmark design. Large-scale studies in recent years have repeatedly found that CatBoost outperforms LightGBM, and LightGBM outperforms XGBoost. So why was the strongest baseline not included?

https://github.com/valeman/Awesome_CatBoost

**Comment by jonaslandsgesell** [comment]
@valeman thank you for your suggestion. 
I will add the CatBoost multiquantile regression today or tomorrow to our benchmark runs. Unfortunately, it will run very slowly due to lacking GPU support https://github.com/catboost/catboost/issues/2783 
I will also add RealMLP https://github.com/dholzmueller/pytabkit as another strong model which is not pretrained.

**Comment by jonaslandsgesell** [comment]
@valeman 
I’ve updated the leaderboard with results from CatBoost multiquantile regression: https://scoringbench.bolt.host/

Interestingly, CatBoost performs very similarly to XGBoost quantile regression, with the notable exception of the CRLS (Exceedance Probability Score). You can find the details on the scoring metric in this paper: https://link.springer.com/article/10.1080/15598608.2012.695663

If you or someone else would like to propose an improved CatboostWrapper ( https://github.com/jonaslandsgesell/ScoringBench/blob/master/scoringbench/wrappers/catboost_wrapper.py ) incorporating CatBoost-specific best practices, please feel free to open a PR! I’m happy to rerun the benchmark with any suggested optimizations.

**Comment by jonaslandsgesell** [comment]
Closing for now

**Comment by valeman** [comment]
Thank you

**Comment by jonaslandsgesell** [comment]
Thank you for the great suggestion
